### DISCUSSION TITLE
Trying the beta version 2

### DISCUSSION THREAD

**Discussion by loctran016** [vuepress-theme-hope/vuepress-theme-hope#616] [Q&A] [answered]
Hi there. As vuepress-next have the package vuepress@next for beta testing, does vuepress-theme-hope have version 2 beta for people to try (I can't use v1 with vuepress@next as there are so many changes which are described in the guide)? I'm interested in the v2 of vuepress-theme-hope so tell me when you publish it. Thanks

**Comment by Mister-Hope** [comment, marked as answer]
Meteorlxy is still making changes around the version. It's stable outside, but not inside. I will ship version after it moves to rc.

Otherwise I have to check through codes and change them.l between versions.

**Reply by loctran016** [reply]
Can't wait for that. Have a good day

**Reply by Mister-Hope** [reply]
I am publishing versions now, but these versions are very unstable, and I can not promise you they will be workable.

**Reply by loctran016** [reply]
How could I try this (How to install, like `yarn add vuepress-theme-hope@next`?)

**Reply by Mister-Hope** [reply]
Yes

**Reply by Mister-Hope** [reply]
Again your are warned, Also a notice, current `next` version is not compatable with vuepress@2.0.0-beta.18 because breakingchanges in beta.18, try it with beta.17.

So take your own risk that any update may break your project.

**Reply by Mister-Hope** [reply]
Both the vuepress and my plugins are not stable enough to try in production env.

**Reply by loctran016** [reply]
Oh, don't worry about it, I only create a new pages using vuepress beta and vp-theme-hope beta just for fun only. Btw, how to install v17???

**Reply by Mister-Hope** [reply]
```sh
yarn add vuepress@2.0.0-beta.17
```

**Reply by loctran016** [reply]
Have I done sth wrong? My repo is [here](https://github.com/Cokaps/vuepressbeta) and my site is [here] as the site look quite weird. Actually there are some contents but after adding sideabar. (This project made for fun so I don't have to be worried about anything)
![image](https://user-images.githubusercontent.com/64778285/121799855-a9a0ad00-cc58-11eb-8c67-827b25987cf8.png)

**Reply by Mister-Hope** [reply]
Actually there is no docs now, so you should read source code about plugin options and use typescript type check to check your theme config.

The themeConfig has changed a lot (followed with theme-default), so v1 config won't work.

And I have plans to change it to a format I prefer, so the api will also change in the future.

**Reply by Mister-Hope** [reply]
For plugin options, please read `packages/<plugin name>/src/shared/options.ts` file.

For theme, emm, I even not finish building it. Nearly half of the features won't work.

You can try with the plugins for now, because the theme is now bundling them. For the theme ifself, maybe you can try around 10 days later.

I am still focusing on the plugins. The theme is just workable for me to test the plugins cooperation.

**Reply by Mister-Hope** [reply]
A big repo like this will take times to migrate.
